internal/config/application.go

```diff
@@ -29,13 +32,15 @@ type parser interface {
 
 // Application is the main syft application configuration.
 type Application struct {
-	ConfigPath         string             `yaml:",omitempty" json:"configPath"`                                                         // the location where the application config was read from (either from -c or discovered while loading)
+	// the location where the application config was read from (either from -c or discovered while loading); default .syft.yaml
+	Config    string `yaml:"config,omitempty" json:"config" mapstructure:"config"`
+	Verbosity uint   `yaml:"verbosity,omitempty" json:"verbosity" mapstructure:"verbosity"`
+	// -q, indicates to not show any status output to stderr (ETUI or logging UI)
+	Quiet              bool               `yaml:"quiet" json:"quiet" mapstructure:"quiet"`
 	Outputs            []string           `yaml:"output" json:"output" mapstructure:"output"`                                           // -o, the format to use for output
 	File               string             `yaml:"file" json:"file" mapstructure:"file"`                                                 // --file, the file to write report output to
-	Quiet              bool               `yaml:"quiet" json:"quiet" mapstructure:"quiet"`                                              // -q, indicates to not show any status output to stderr (ETUI or logging UI)
 	CheckForAppUpdate  bool               `yaml:"check-for-app-update" json:"check-for-app-update" mapstructure:"check-for-app-update"` // whether to check for an application update on start up or not
 	Anchore            anchore            `yaml:"anchore" json:"anchore" mapstructure:"anchore"`                                        // options for interacting with Anchore Engine/Enterprise
-	CliOptions         CliOnlyOptions     `yaml:"-" json:"-"`                                                                           // all options only available through the CLI (not via env vars or config)
```

Comment:
TODO: There is a small conflict here between the input as indicated by the user and the translation used for the application. EX: `scope string ==> scope internal usage` `verbosity uint ==> verbosity logging config`

Maybe we can reconcile this